test/resource_test.rb

```diff
@@ -119,13 +119,17 @@ def test_acquire_with_fork
     resource.acquire do
       fork do
         resource.acquire do
-          assert_raises Semian::TimeoutError do
```

Comment:
If you don't believe me, do 'assert_equal(true, false)' here, and you will see the test still passes.